_config/email.yml

```diff
@@ -1,12 +1,55 @@
 ---
-Name: emailconfig
+Name: mailer
 ---
 SilverStripe\Core\Injector\Injector:
-  Swift_Transport: Swift_MailTransport
-  Swift_Mailer:
-    constructor:
-      - '%$Swift_Transport'
-  SilverStripe\Control\Email\Mailer:
-    class: SilverStripe\Control\Email\SwiftMailer
-    properties:
-      SwiftMailer: '%$Swift_Mailer'
+  Symfony\Component\Mailer\MailerInterface:
+    class: Symfony\Component\Mailer\Mailer
+    constructor:
+      0: '%$Symfony\Component\Mailer\Transport\TransportInterface'
+      1: '%$Symfony\Component\Messenger\MessageBusInterface.mailer'
```

Comment:
Why pass a bus in here, when we can just pass in the event dispatcher directly? What does the bus do for us that the dispatcher doesn't?
```yml
Symfony\Component\Mailer\MailerInterface:
    class: Symfony\Component\Mailer\Mailer
    constructor:
      transport: '%$Symfony\Component\Mailer\Transport\TransportInterface'
      dispatcher: '%$Symfony\Component\EventDispatcher\EventDispatcherInterface.mailer'
```

For that matter, it looks like the transports (which subclass `AbstractTransport`) all take a dispatcher as an argument to _their_ constructor... so what does any of this setup with the bus, middleware, senderslocator, etc do that just passing `%$Symfony\Component\EventDispatcher\EventDispatcherInterface.mailer` to the transport constructor won't achieve?

Reply:
That's correct you can, I made a bad assumption when doing this that the a bus is required for the dispatcher to get called properly, though the dispatcher is called in AbstractTransport::send() if no bus is provided.  I've updated code accordingly, including removing the symfony/messanger component entirely